src/OpenLoco/Ui/WindowManager.cpp

```diff
@@ -35,7 +35,7 @@ namespace OpenLoco::Ui::WindowManager
 
     constexpr size_t max_windows = 12;
 
-    static loco_global<uint16_t, 0x0050C19C> time_since_last_tick;
+    static loco_global<uint16_t, 0x0050C19C> timeSinceLastTick;
```

Comment:
Prefix with _

Reply:
apologies for missing this. I also noticed a missing underscore on `gCurrentRotation` a few lines down from here, I added it there too